timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -201,9 +82,8 @@ def __init__(
 
         self.verbosity = verbosity
 
-        # Dict of normal model -> FULL model. FULL models are produced by
-        # self.refit_single_full() and self.refit_full().
-        # TODO: reconcile this with abstract classes
+        #: Dict of normal model -> FULL model. FULL models are produced by
+        #: self.refit_single_full() and self.refit_full().
```

Comment:
`#:` is a typo?

Reply:
apparently this is a [doc comment](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#doc-comments-and-docstrings) that I thought we might start using for the non-public stuff (will remove it from the other PR)

Reply:
TIL! Are there some cases in which Sphinx will build the documentation for the Trainer class?

Reply:
Not really, but we still follow sphinx conventions.